index.html

```diff
@@ -135,7 +315,120 @@ <h2>Example Modal 2</h2>
 
     // JavaScript Support Example
     DCFUtility.flagSupportsJavaScript();
-    </script>
+
+    const keyframesClose1 = [
```

Comment:
Why is this here?  Isn't this in DCF `dcf-slideshow-theme.js`

Reply:
Ryan says that dcf-slideshow-theme is something that a developer will customize for his/her own theme so it should be in dcf_starter and eventually wdn.

Reply:
ok

Reply:
This is fine but we'll need to make sure the dcf placeholder javascript `dcf-slideshow-theme.js` and sample here in dcf_starter stay in sync or just manage it in dcf_starter some how.  We would need to at least lint check it if only in starter.